supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--a.adoc

```diff
@@ -0,0 +1,23 @@
+= A
+// Should these terms be added to the AMQ conventions.
+[discrete]
+=== ActiveMQ
+
+*Description* Shorthand for "ActiveMQ Artemis," the built-in messaging technology for JBoss EAP.
```

Comment:
I think the convention is to use a complete sentence for the Description.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.